compiler/src/iree/compiler/Dialect/HAL/Target/CUDA/CUDATarget.cpp

```diff
@@ -47,11 +55,165 @@ static llvm::cl::opt<std::string> clTargetChip(
     "iree-hal-cuda-llvm-target-arch", llvm::cl::desc("LLVM target chip."),
     llvm::cl::init("sm_35"));
 
+static llvm::cl::opt<bool> clUsePtxas(
+    "iree-hal-cuda-use-ptxas", llvm::cl::init(true),
+    llvm::cl::desc(
+        "It uses the default ptxas compiler, compiles the generated PTX "
+        "code with it, puts the cubin binary generated by ptxas into the "
+        "executable. '--iree-hal-cuda-llvm-target-arch' is used as the "
+        "target GPU. If passing additional parameters to ptxas is desired, the "
+        "parameters flag can be used "
+        "(e.g.'--iree-hal-cuda-use-ptxas-params=-v')."));
```

Comment:
`--iree-hal-use-ptxas` "It uses the default ptxas compiler." 
Is default ptxas compiler means using driver's JIT ptxas flow? What does default mean here?

From the PR description:
```
When --iree-hal-use-ptxas=false, packs the ptx into vmfb file, and let the runtime to compile.
Set--iree-hal-use-ptxas to true. So default compilation is ptxas.
```

it seems like `--iree-hal-use-ptxas=false` is the current behaviour of iree-compile, i.e., ptx in vmfb file is JIT compiled to SASS by the driver. 

`--iree-hal-use-ptxas=true` uses ptxas provided by `--iree-hal-cuda-use-ptxas-from`

Should the default setting for --iree-hal-use-ptxas be false setting it to the status quo? Let me know if I am missing something here.

Reply:
yes should be changed to `false`

Reply:
I clarified the text here, `--iree-hal-use-ptxas` finds `ptxas` on your PATH. 
I changed the default to `--iree-hal-use-ptxas=false`. 

as a future work, we could make default compilation `cubin` + `ptx`. If `cubin` is not for the target, we jit `ptx` on the runtime.